api/Notification.json

```diff
@@ -824,6 +824,39 @@
           }
         }
       },
+      "navigate": {
+        "__compat": {
+          "spec_url": "https://notifications.spec.whatwg.org/#dom-notification-navigate",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": "18.4"
```

Comment:
Shipped via https://github.com/WebKit/WebKit/commit/a786946331264a29f40aca673a1953d573fa60d7 in WebKit [621.1.9](https://github.com/WebKit/WebKit/blob/a786946331264a29f40aca673a1953d573fa60d7/Configurations/Version.xcconfig#L24-L26) before Safari [18.4](https://github.com/mdn/browser-compat-data/blob/e16cbd7307944ab9cbfd4ee9a544975ca3d919ea/browsers/safari.json#L349-L355).